### DISCUSSION TITLE
HTMLによるタスクリストの記述

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#340] [要望・アイデア・不具合]
HTML でタスクリストを次のように記述すると、チェックボックスが期待したとおりにレンダリングされます。これは意図した挙動でしょうか？

![image](https://user-images.githubusercontent.com/75878492/197729077-ed9e1081-1fbd-4246-b8d5-b4d1208ca715.png)

<details><summary>サンプルコード</summary>

```html
<ul>
    <li>[ ] チェックなし</li>
    <li>[x] チェックあり</li>
    <li>[ ] チェックなし
        <ul>
            <li>[x] チェックあり</li>
        </ul>
    </li>
</ul>
```

</details>

`<input>` タグを利用すればチェックボックスを表示させることができるかもしれませんが、こちらの方が簡便です。意図した挙動でなかったとしても、この記述の方法があると Markdown 表内にタスクリストを含める際に楽なので、残しておいてほしいと思っています。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご報告ありがとうございます！
意図していた挙動ではありませんが、
現時点では無くす予定はありませんので、そのままお使いください✨

**Reply by yarakos95** [reply]
分かりました。ありがとうございます！
